### ISSUE TITLE
Implement showprofile intent to display validator's profile

### ISSUE THREAD

**Issue by wakqasahmed** [tuum-tech/vouch-capsule#82]
In not so distant future, we can implement 'show profile' intent for the user to make sure they are choosing the right validator, as there could be many people registered as a validator with the same first and last name, imagine professors of different universities validating their students kind of like proof of study consensus :)

**Comment by kpachhai** [comment]
Yes, this is actually one of the first use case I was also thinking about implementing on vouch app.

**Comment by mfaizanatiq** [comment]
Could there be a case where professors and validators gives biased verifications. For e.g Pay the validator and then get your self verified.

Could be totally a hypothetical scenario but just thinking.

**Comment by kpachhai** [comment]
yea that can happen. but we cannot really prevent something like that from happening. Unless you only assign one dedicated account that can verify your education and such

**Comment by mfaizanatiq** [comment]
When I was designing the ’become a validator’ user flow screens, this is one thing I was thinking.

In the current flow, we allow everyone to onboard to become a validator with ’Become a validator’ button on the top, how will we verify that validator credibility?

**Comment by kpachhai** [comment]
We can't and that's ok. This is in the spirit of decentralization. But those validators that register this way will have less priority compared to the validators we have added ourselves as Tuum Tech(these will be what we'll suggest as preferred ones)
